src/features/Auth/pages/ResetPasswordPage.tsx

```diff
@@ -0,0 +1,80 @@
+import { UserOutlined, LockOutlined } from '@ant-design/icons'
+import { Flex, Form, Input, Typography, message } from 'antd'
+import { useNavigate } from 'react-router-dom'
+import { BaseButton } from '../../../components/ui'
+import { ROUTE_PATH } from '../../../constants/routePath'
+
+type ResetPasswordForm = {
+  newPassword: string
+  confirmPassword: string
+}
+
+function ResetPasswordPage() {
+  const [form] = Form.useForm<ResetPasswordForm>()
+  const navigate = useNavigate()
+  const [messageApi, contextHolder] = message.useMessage()
+
+  const onFinish = (values: ResetPasswordForm) => {
+    console.log('Submitted:', values)
+    // Implement your logic here
+    messageApi.success('Mật khẩu đã được đặt lại thành công.')
```

Comment:
chỗ này vì các câu lệnh không dùng bất đồng bộ nên nó bị chuyển qua màn hình Login chứ không có thông báo
sửa như sau
  
```suggestion
  const onFinish = async (values: ResetPasswordForm) => {
    console.log('Submitted:', values)
    // Implement your logic here
    await messageApi.success('Mật khẩu đã được đặt lại thành công.')
    navigate(ROUTE_PATH.LOGIN)
  }
   
```